pages/rooms/[roomId].js

```diff
@@ -0,0 +1,54 @@
+import { useRouter } from 'next/router';
+import styled from 'styled-components';
+import BackButton from '../../components/BackButton';
+import DeskItem from '../../components/DeskItem';
+import Header from '../../components/Header';
+import { getAllRooms, getRoomById } from '../../services/roomService';
+
+export async function getStaticPaths() {
+  const allRooms = await getAllRooms();
+  const paths = allRooms.map((rooms) => {
+    return { params: { roomId: rooms.id } };
+  });
+
+  return {
+    paths,
+    fallback: false,
+  };
+}
+
+export async function getStaticProps(context) {
+  const roomId = context.params.roomId;
+  const roomDetails = await getRoomById(roomId);
+
+  return {
+    props: { roomDetails },
+  };
+}
+
+export default function RoomPage({ roomDetails }) {
+  const router = useRouter();
+  const { roomId } = router.query;
```

Comment:
```suggestion
```
this variable is unused and can be deleted, because your roomDetails prop already contains all the information you use afterwards (which are generated by getStaticProps)

Reply:
You are right. Totally oversaw that. Thanks for the hint!